gensim/models/phrases.py

```diff
@@ -590,10 +589,11 @@ def _learn_vocab(sentences, max_vocab_size, delimiter, connector_words, progress
             start_token, in_between = None, []
             for word in sentence:
                 if word not in connector_words:
-                    vocab[word] += 1
+                    vocab[word] = vocab.get(word, 0)+1
```

Comment:
Space around binary operators (here and everywhere):

```suggestion
                    vocab[word] = vocab.get(word, 0) + 1
```